iron-menu-behavior.html

```diff
@@ -101,7 +107,11 @@
       var selectedItem = this.multi ? (this.selectedItems && this.selectedItems[0]) : this.selectedItem;
 
       this.items.forEach(function(item) {
-        item.setAttribute('tabindex', item === selectedItem ? '0' : '-1');
+        // Don't enable tab index if an item is disabled, even if it's selected.
+        if (item === selectedItem && !selectedItem.hasAttribute('disabled') && !this.disabled)
+          item.setAttribute('tabindex', 0);
+        else
+          item.removeAttribute('tabindex');
```

Comment:
Ok, talked with @alice yesterday about this, seems like removing tabindex is the best option.